### PR TITLE
fix: :bug: uncommented openCollapse

### PR DESCRIPTION
**Pull request by Edithmark42**
uncommented openCollapse to resolve the collapse not working issue

Ref #103

### CODE REVIEW

File: package/utils/menuUtil.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ export function createMultiLevelMenu(
         {multiLevel &&
           itemData?.Children &&
           itemData?.Children.length > 0 &&
-          // openCollapse && Object.keys(openCollapse).length > 0 && openCollapse[itemData.id] &&
+          openCollapse && Object.keys(openCollapse).length > 0 && openCollapse[itemData.id] &&
```
Comment (techoneel) [inline package/utils/menuUtil.js:61]: need to check whether the key exists.
suggestion: 
```
Object.keys(openCollapse).includes(itemData?.id)
```

**Review by techoneel — CHANGES_REQUESTED** [review]
Please fix all the issues identified

**Review by techoneel — APPROVED** [review]
Good to go

File: package/utils/menuUtil.js (hunk 1 above)
Comment (Edithmark42) [reply package/utils/menuUtil.js]: done

**Review by techoneel — APPROVED** [review]
Good to go

**Pull request merged**